Makefile

```diff
@@ -132,6 +132,22 @@ MIPS_VERSION := -mips2
 # we support Microsoft extensions such as anonymous structs, which the compiler does support but warns for their usage. Suppress the warnings with -woff.
 CFLAGS += -G 0 -non_shared -fullwarn -verbose -Xcpluscomm $(IINC) -nostdinc -Wab,-r4300_mul -woff 649,838,712,516
 
+# The system heap is an area of memory that covers all the VRAM that is not
+# used by compiled code/data.
```

Comment:
```suggestion
# The system heap is an area of memory that covers all the RAM that is not
# taken by compiled code/data.
```
vram -> ram: because either "vram" as in the linker view of memory includes >8MiB, yet the heap is a very concrete area so may as well not confuse it with it,  or "vram" as in virtual refering to the cpu address space being irrelevant in the context

used -> taken: since the heap is "used" by code too, I think you meant the bytes that the various code and data sections are made of, saying "taken" for that seems less confusing